src/dtclient/kubernetes_settings.go

```diff
@@ -133,6 +118,57 @@ func (dtc *dynatraceClient) CreateOrUpdateKubernetesSetting(clusterLabel, kubeSy
 	return resDataJson[0].ObjectId, nil
 }
 
+func (dtc *dynatraceClient) getKubernetesSettingBody(clusterLabel, kubeSystemUUID, scope string, schemaId string, isK8sHierarchicalMonitoringSettings bool) []postKubernetesSettingsBody {
```

Comment:
does not need to be a method

Reply:
for me it is question of taste, additionally I can test response body separately as json 
src/dtclient/kubernetes_settings_test.go:303
please let me know if we should really remove this method, thanks

Reply:
```suggestion
func getKubernetesSettingBody(clusterLabel, kubeSystemUUID, scope string, schemaId string, isK8sHierarchicalMonitoringSettings bool) []postKubernetesSettingsBody {
```
I mean to change it to a function, not completely remove it.

Reply:
done